fzf-marks.plugin.bash

```diff
@@ -171,18 +171,26 @@ function set-up-fzm-bindings {
     elif [ "${BASH_VERSINFO[0]}" -lt 4 ]; then
         bind "\"\C-g\":\"fzm\C-m\""
     else
-        local mark1='\200' mark2='\201' mark3='\202'
-
-        # Change markers for UTF-8 encodings
+        # Intiialize special keys used for key bindings
+        _fzm_key1='\200'
+        _fzm_key2='\201'
+        _fzm_key3='\202'
```

Comment:
Here I chose `_fzm_*` as the prefix for the private global variables used by `fzf-marks`. If you have any suggestion on the naming convention on the private variables, please let me know.

Reply:
Seems clean to me!